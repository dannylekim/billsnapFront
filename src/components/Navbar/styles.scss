.navbar {
  width: 100%;
  height: fit-content;
  font-size: 20px;
  font-weight: bold;
  // margin-bottom: -100rem; this will stay commented out until someone writes why we need it
}
