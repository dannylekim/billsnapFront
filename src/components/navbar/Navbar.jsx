import React from 'react';
import './styles.scss';
import {Nav, NavItem, NavLink} from "shards-react";
import {FaBars, FaHome, FaMoneyBillWave} from 'react-icons/fa';
// FaBars FaHome FaMoneyBillWave
export default () => {
    return (
        <Nav tabs>
            <NavItem>
                <NavLink href="#">
                    <FaBars/>
                </NavLink>
            </NavItem>
            <NavItem>
                <NavLink href="#">
                    <FaHome/>
                </NavLink>
            </NavItem>
            <NavItem>
                <NavLink>
                    <FaMoneyBillWave/>
                </NavLink>
            </NavItem>
        </Nav>
    )
}