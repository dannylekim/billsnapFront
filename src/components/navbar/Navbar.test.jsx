import React from 'react';
import Navbar from './Navbar.jsx';

describe('LoginPage', () => {
    let wrapper;
    let handleMockFunction;

    beforeEach(() => {
        handleMockFunction = jest.fn();
        wrapper = shallow(<Navbar/>);
    })

    describe('render', () => {
        describe('snapshots ðŸ“¸', () => {
            it('LoginPage should match snap shot', () => {
                matches(<Navbar/>);
            });
        })
    })
})