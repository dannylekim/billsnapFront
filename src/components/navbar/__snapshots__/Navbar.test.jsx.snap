// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage render snapshots ðŸ“¸ LoginPage should match snap shot 1`] = `
<ul
  className="nav nav-tabs"
>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="#"
      onClick={[Function]}
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
        />
      </svg>
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="#"
      onClick={[Function]}
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 576 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
        />
      </svg>
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      onClick={[Function]}
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 640 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M621.16 54.46C582.37 38.19 543.55 32 504.75 32c-123.17-.01-246.33 62.34-369.5 62.34-30.89 0-61.76-3.92-92.65-13.72-3.47-1.1-6.95-1.62-10.35-1.62C15.04 79 0 92.32 0 110.81v317.26c0 12.63 7.23 24.6 18.84 29.46C57.63 473.81 96.45 480 135.25 480c123.17 0 246.34-62.35 369.51-62.35 30.89 0 61.76 3.92 92.65 13.72 3.47 1.1 6.95 1.62 10.35 1.62 17.21 0 32.25-13.32 32.25-31.81V83.93c-.01-12.64-7.24-24.6-18.85-29.47zM48 132.22c20.12 5.04 41.12 7.57 62.72 8.93C104.84 170.54 79 192.69 48 192.69v-60.47zm0 285v-47.78c34.37 0 62.18 27.27 63.71 61.4-22.53-1.81-43.59-6.31-63.71-13.62zM320 352c-44.19 0-80-42.99-80-96 0-53.02 35.82-96 80-96s80 42.98 80 96c0 53.03-35.83 96-80 96zm272 27.78c-17.52-4.39-35.71-6.85-54.32-8.44 5.87-26.08 27.5-45.88 54.32-49.28v57.72zm0-236.11c-30.89-3.91-54.86-29.7-55.81-61.55 19.54 2.17 38.09 6.23 55.81 12.66v48.89z"
        />
      </svg>
    </a>
  </li>
</ul>
`;
